--- rustc-1.41.0-src/src/bootstrap/builder.rs.orig      2020-06-11 16:46:49.269083808 +0000
+++ rustc-1.41.0-src/src/bootstrap/builder.rs   2020-06-11 16:48:08.438500190 +0000
@@ -884,8 +884,6 @@
             rustflags.arg("-Zforce-unstable-if-unmarked");
         }
 
-        rustflags.arg("-Zexternal-macro-backtrace");
-
         let want_rustdoc = self.doc_tests != DocTests::No;
 
         // We synthetically interpret a stage0 compiler used to build tools as a
@@ -970,7 +968,7 @@
                 rustflags.arg("-Zosx-rpath-install-name");
                 Some("-Wl,-rpath,@loader_path/../lib")
             } else if !target.contains("windows") {
-                Some("-Wl,-rpath,$ORIGIN/../lib")
+                Some("-Wl,-rpath,/usr/clang/9.0/lib")
             } else {
                 None
             };
