--- rustc-1.44.0-src/src/bootstrap/bootstrap.py.orig    2020-06-10 10:19:06.267938804 +0000
+++ rustc-1.44.0-src/src/bootstrap/bootstrap.py 2020-06-10 10:19:30.009384961 +0000
@@ -682,7 +682,7 @@
             if "LIBRARY_PATH" in env else ""
         # preserve existing RUSTFLAGS
         env.setdefault("RUSTFLAGS", "")
-        env["RUSTFLAGS"] += " -Cdebuginfo=2"
+        env["RUSTFLAGS"] += " -Copt-level=2"
 
         build_section = "target.{}".format(self.build_triple())
         target_features = []
