diff --git a/record/record.c b/record/record.c
index 018e53f..cf33db8 100644
--- a/record/record.c
+++ b/record/record.c
@@ -1404,6 +1404,9 @@ typedef struct {
 #undef ERR /* Avoid conflict with Solaris <sys/regset.h> */
 #endif
 
+#undef ERR /* protect against #define ERR in <sys/regset.h> on Solaris x86,
+	      which gets included indirectly via #include <stdlib.h> */
+
 /* These constant are used to index into an array of SetInfoRec. */
 enum { REQ,                     /* set info for requests */
     REP,                        /* set info for replies */
